src/Transports.AspNetCore/WebSockets/GraphQLWs/SubscriptionServer.cs

```diff
@@ -0,0 +1,261 @@
+using GraphQL.Transport;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets.GraphQLWs;
+
+/// <inheritdoc/>
+public class SubscriptionServer : BaseSubscriptionServer
+{
+    private readonly IWebSocketAuthenticationService? _authenticationService;
+
+    /// <summary>
+    /// The WebSocket sub-protocol used for this protocol.
+    /// </summary>
+    /// <remarks>
+    /// Please note that the correct sub-protocol for the
+    /// <see href="https://github.com/enisdenjo/graphql-ws">graphql-ws</see>
+    /// protocol is <c>graphql-transport-ws</c> as is defined here.
+    /// </remarks>
+    public const string SubProtocol = "graphql-transport-ws";
```

Comment:
Constants use `SUB_PROTOCOL` naming conventions.

Reply:
If so then this section in editorconfig should be changed/removed
```
########## name all constant fields using UPPER_CASE ##########
# https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-naming-conventions?view=vs-2019
# dotnet_naming_rule.<namingRuleTitle>.symbols = <symbolTitle>
dotnet_naming_rule.constant_fields_should_be_upper_case.symbols = constant_fields

# dotnet_naming_symbols.<symbolTitle>.<property> = <value>
dotnet_naming_symbols.constant_fields.applicable_kinds = field
dotnet_naming_symbols.constant_fields.applicable_accessibilities = *
dotnet_naming_symbols.constant_fields.required_modifiers = const

# dotnet_naming_rule.<namingRuleTitle>.style = <styleTitle>
dotnet_naming_rule.constant_fields_should_be_upper_case.style = upper_case_style

# dotnet_naming_style.<styleTitle>.<property> = <value>
dotnet_naming_style.upper_case_style.capitalization = all_upper
dotnet_naming_style.upper_case_style.word_separator = _

# dotnet_naming_rule.<namingRuleTitle>.severity = <value>
dotnet_naming_rule.constant_fields_should_be_upper_case.severity = warning
```

We used this convention for a long time.

Reply:
Yeah, I updated the editorconfig for my project to match the code, and have many other changes in there.

In any case, what do you think -- should we match .NET here or not?  Excluding the MessageType constants, the two `SubProtocol` constants are the only public constants.

Reply:
I'm used to see CONSTANT_CASE for constants. Such a notation allows you to distinguish constants from properties. In addition, when I added rule for such a notation, I was guided not by my preferences, but by some recommendations, which, however, I no longer remember.

Reply:
So I would change all constants to CONST_CASE.

Reply:
What if we just make it a static property?

Reply:
(I dislike constant case.)

Reply:
static property vs const is a eternal battle. Not a big deal to use static (+readonly) property/field instead of const.

Reply:
I'll do that then, since I don't want to use constant case.

Reply:
const value is embedded within caller side, property - not